auctioneer/client.go

```diff
@@ -781,6 +781,12 @@ func (c *Client) wait(backoff time.Duration) error {
 	}
 }
 
+// IsSubscribed returns true if at least one account is in an active state and
+// the subscription stream to the server was established successfully.
+func (c *Client) IsSubscribed() bool {
+	return c.serverStream != nil
```

Comment:
need mutex?

Reply:
Ah yes. There even was a mutex already. Fixed another potentially unguarded access by narrowing the lock: 8a308dd8